### PR TITLE
libtasn1

### PR DESCRIPTION
**Pull request by Ludea**

### CODE REVIEW

File: packages/libtasn1/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/libtasn1/
+TERMUX_PKG_DESCRIPTION="This is GNU Libtasn1, a small ASN.1 library"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_VERSION=4_16_0
```
Comment (xeffyr) [inline packages/libtasn1/build.sh:4]: Wrong version name. Should be separated with `.` (dot).

File: packages/libtasn1/build.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+TERMUX_PKG_SRCURL=https://gitlab.com/gnutls/libtasn1/-/archive/libtasn1_${TERMUX_PKG_VERSION}/libtasn1-libtasn1_${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=2d1b0cbd26edfcb54694b2339106a02a81d630a7dedc357461aeb186874cc7c0
+TERMUX_PKG_BUILD_DEPENDS="texinfo, help2man"
+
+termux_step_post_get_source {
```
Comment (xeffyr) [inline packages/libtasn1/build.sh:9]: I don't think this is a right syntax for Bash function definition.

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by Ludea** [comment]
seems ci doesn't find gtk-doc

**Comment by xeffyr** [comment]
> TERMUX_PKG_BUILD_DEPENDS="git, gtk-doc, texinfo, help2man"

This will not work. Remember that Termux package build environment is not Termux. That's Ubuntu 20.04 (amd64) installation. You can't use here tools packaged for Termux.

Remove `TERMUX_PKG_BUILD_DEPENDS` completely. It is used only for things like static libraries.

**Pull request merged**